### DISCUSSION TITLE
Export plotnine plot as a matplotlib one

### DISCUSSION THREAD

**Discussion by lucazav** [has2k1/plotnine#625] [Q&A]
I'm wondering if it's possible to export (or just pickle) a _plotnine_ plot as _matplotlib_ one, so that I can unpickle and plot it just importing the _matplotlib_ library (without importing _plotnine_).

Thank you.

**Comment by lucazav** [comment]
Any plotnine expert that could help on this, please?

**Reply by has2k1** [reply]
Sorry I did not see this earlier. 

In my experience, pickling matplotlib objects is fickle, has edge cases and tends to break with updates. So depending on what version of MPL, this may or may not work.

```python
import pickle

p = ggplot() + ...
fig = p.draw()

pickle.dump(fig, 'fig.pkl')
```

**Reply by lucazav** [reply]
Thank you @has2k1! Do you know from what version of MPL it works for sure? Thanks
